docs/tutorial/working-with-tables/index.md

```diff
@@ -30,6 +30,20 @@ first import it into the sandbox.
 >>> from ehrql.tables.beta.core import patients
 ```
 
+??? tip "Importing tables into the sandbox"
+    Importing a table into the sandbox doesn't display any output.
+    In other words, typing
+
+    ```pycon
+    >>> from ehrql.tables.beta.core import patients
+    ```
+
+    into the sandbox and pressing ++enter++ displays
+
+    ```pycon
+    >>>
+    ```
```

Comment:
In the interests of keeping things snappy, do we need the code sample here?

Reply:
Catherine spent a couple of minutes clicking around after typing

```py
>>> from ehrql.tables.beta.core import patients
```

into the sandbox and pressing enter, and ended up opening *example-data/patients.csv*. Whilst I agree this snippet isn't snappy, I wanted to be extra extra sure that the user didn't expect any output. Consequently, I'd argue that we do need it.

Reply:
Won't that just be served by saying

> Importing a table into the sandbox doesn't display any output.

?

Don't feel strongly though!